src/entry.rs

```diff
@@ -309,6 +320,29 @@ impl<'a> EntryFields<'a> {
                                         for `{}`", mode, dst.display()), e)
             }));
         }
+        // Windows does not completely support posix xattrs
+        // https://en.wikipedia.org/wiki/Extended_file_attributes#Windows_NT
+        if cfg!(target_family = "unix") && preserve_xattrs {
```

Comment:
Is there a downside from doing this by default? May be worth seeing if `tar` does that by default